### DISCUSSION TITLE
Ability to remove overridden parameters to a certain app without knowing which parameters

### DISCUSSION THREAD

**Discussion by cristi-vlad** [argoproj/argo-cd#8454] [Ideas]
By using **argocd app unset** you need to know exactly which parameter you want to unset but maybe there is worth to have something like hard reset of an application using argocd cli (to be used into scripts)
Also, I couldn't find a way by using **argocd app unset** to group apps based on labels and unset all (whether or not they have rewritten parameters)

**Comment by RomanBats** [comment]
You may try using "argocd app patch" to remove whole .spec.source.helm.parameters node.

**Comment by FrittenToni** [comment]
Just for reference. This command did the trick for me: argocd app patch [appName] --patch '{"spec": { "source": { "helm": { "parameters": [] }}}}' --type merge

**Reply by cristi-vlad** [reply]
Thank you !
